code/style.css

```diff
@@ -1,4 +1,213 @@
-html {
-  background: #0025ff;
-  color: #fff;
+body {
+  margin: 0;
+  background: white
```

Comment:
if you put your font-family in the body you don´t need to declare it for all headings and so on and you save yourself some rows of coding :)